rollup.config.ts

```diff
@@ -144,4 +146,24 @@ const buildNative = {
   ],
 }
 
-export default [buildNode, buildNative, buildEsm, buildUdm]
+/**
+ * Configuration for the iife build
+ */
+const buildIife = {
+  input: 'src/index.ts',
+  output: {
+    file: 'lib/iife/index.js',
+    name: 'MockServiceWorker',
+    format: 'iife',
+    esModule: false,
+  },
+  plugins: [
+    replace({
+      'process.env.NODE_ENV': JSON.stringify('development'),
```

Comment:
Could you please remind me why IIFE module should be bundled for the "development" environment?

Reply:
Out of an abundance of caution, which now that I am revisiting this does not seem necessary.

In development mode, there is an extra check resulting in an error warning "when multi-realm constructors are detected", you can see it in the compiled UMD version. So by the time we get to a production ready bundle, warnings should have been identified and taken care of. I'll change this to production.